packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -249,6 +257,50 @@ type Props = {
     itemsMap?: ItemsMap;
 };
 
+const validateMsTeamsWebhook = (webhook: string): boolean => {
+    if (webhook.length === 0) return false;
+    if (!webhook.startsWith('https://')) return false;
+    if (/\s/.test(webhook)) return false;
+
+    return true;
+};
+
+type MicrosoftTeamsDestinationProps = {
+    form: AnyType;
+};
+const MicrosoftTeamsDestination: FC<MicrosoftTeamsDestinationProps> = ({
+    form,
```

Comment:
instead of passing the prop `form` you can just have 2 props: `msTeamsTargets` and `onChange` and handle these in the parent